README.md

```diff
@@ -39,7 +39,7 @@ If any are found that do not match the SHA-256 checksums of our official release
 Simply add this action to your workflow **after** having checked out your source tree and **before** running any Gradle build:  
 
 ```yaml
-uses: gradle/wrapper-validation-action@v1
+uses: gradle/wrapper-validation-action@releases/v1
```

Comment:
:x: why change this to point to a branch instead of a release tag ?

Reply:
That was unintentional and not part of the change I meant to make.